signal_analog/dashboards.py

```diff
@@ -459,9 +412,8 @@ def get_config_helper(id):
                     'column': column_num_to_set,
                     'height': 2,
                     'row': row_num_to_set,
-                    'width': 6
-                })
-        return state
+                    'width': 6})
+            return state
```

Comment:
It looks like the indentation was modified here that appears to change meaning of this code. As I read it right now we're returning as soon as we find a chart that is not in the set of remote dashboards, which means that we're not guaranteed to process all charts in the dashboard. Is that intentional?

Reply:
Nope, that was not intended at all. Fixed it. Excellent catch!